cloud/jenkins/pxc_containers_docker_build.groovy

```diff
@@ -197,13 +265,15 @@ pipeline {
     }
     post {
         always {
-            archiveArtifacts artifacts: '*.log', allowEmptyArchive: true
             archiveArtifacts artifacts: '*.pdf', allowEmptyArchive: true
             sh '''
                 sudo docker rmi -f \$(sudo docker images -q) || true
             '''
             deleteDir()
         }
+        unstable {
+            slackSend channel: '#cloud-dev-ci', color: '#F6F930', message: "Building of PXC docker images unstable. Please check the log ${BUILD_URL}"
+        }
         failure {
```

Comment:
As I see we have the same message. Could we just put test result to the message? Could we send notification to the person who run build (if it was run by some person) ?

Reply:
@nmarukovich it is not the same message 'in this case we have unstable'. We can do everything  via separate PR especial if the owner of this task @nmarukovich   :)